articles/quickstart/spa/_includes/_authz_protect_client_side_routes.md

```diff
@@ -0,0 +1,3 @@
+## Protect Client-Side Routes
+
+You may want to give access to some routes in your application only to authenticated users. You can check if the user is authenticated with the ${check}.
```

Comment:
As discussed on slack, this is a placeholder that is replaced when the snippet is inserted into an article (the variable is defined in the insert command).